### PR TITLE
Document getOwner and setOwner

### PR DESCRIPTION
**Pull request by mixonic**
Normally I would include a preview here, but I cannot seem to get a preview of the docs locally :-/

### CODE REVIEW

File: packages/container/lib/owner.js [hunk 1]
```diff
@@ -1,11 +1,64 @@
...
 import { symbol } from 'ember-metal/utils';
 
 export const OWNER = symbol('OWNER');
 
+
+/**
+  Framework objects in an Ember application (components, services, routes, etc.)
+  are created via a factory and dependency injection system.
+
+  `getOwner` fetches the owner object responsible for a instance. This can
```
Comment (twokul) [inline packages/container/lib/owner.js]: an instance?

Comment (mixonic) [inline packages/container/lib/owner.js]: gracias

File: packages/container/lib/owner.js [hunk 2]
```diff
@@ -1,11 +1,64 @@
...
+  @return {Object} an owner object.
+  @for Ember
+  @public
+*/
 export function getOwner(object) {
   return object[OWNER];
 }
 
+/**
+  `setOwner` forces a new owner on a given object instance. This is primarially
```
Comment (dgeb) [inline packages/container/lib/owner.js]: s/primarially/primarily/

File: packages/container/lib/owner.js [hunk 3]
```diff
@@ -1,11 +1,64 @@
...
+  ```
+  // app/components/play-audio.js
+  import Ember from 'ember';
+
+  // Usage:
+  //
+  //   {{play-audio audioType=model.audioType audioFile=model.file}}
+  //
+  export default Ember.Component.extend({
+    audioService: Ember.computed('audioType', function(){
```
Comment (dgeb) [inline packages/container/lib/owner.js]: space needed in `function(){`

File: packages/container/lib/owner.js [hunk 4]
```diff
@@ -1,11 +1,64 @@
...
+
 import { symbol } from 'ember-metal/utils';
 
 export const OWNER = symbol('OWNER');
 
+
+/**
+  Framework objects in an Ember application (components, services, routes, etc.)
+  are created via a factory and dependency injection system.
+
```
Comment (dgeb) [inline packages/container/lib/owner.js]: I feel like we need a general explanation of "owner". Perhaps something like:

> Every framework object is the responsibility of an "owner", which handles its instantiation and manages its lifetime.

**Comment by mixonic** [comment]
Thanks @dgeb, updated.

**Comment by dgeb** [comment]
Thank _you_ @mixonic for tackling these docs. LGTM :+1:

**Pull request merged**